py/tests/conftest.py

```diff
@@ -21,11 +21,11 @@ def pytest_addoption(parser):
     parser.addoption("--dist", action="store", default="")
     parser.addoption("--spark_conf", action="store", default="")
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="session")
```

Comment:
As we discussed, we need to make sure that in client-less approach we can connect to a new backenda

Reply:
Yup, will cover in separated tests